tools/README.md

```diff
@@ -1 +1,20 @@
-Command line utilities for processing and converting 3D Tiles tilesets.
\ No newline at end of file
+# 3D Tiles Tools
+
+Command line utilities for processing and converting 3D Tiles tilesets.
+
+## Instructions
+
+Clone this repo and install [Node.js](http://nodejs.org/).  From the root directory of this repo, run:
+```
+npm install
+```
+
+## Tools
+
+### gzipTileset
+
+Detects whether the tileset is compressed or not and does the opposite.
```

Comment:
Are you sure this is the right form factor, it sounds awkward.

* I thought we were going to have one processing pipeline and be able to build a series of stages in exactly the same way as we plan to do so for gltf-pipeline.  Instead, this looks like the plan is to make lots of separate tools.  Are we sure that is the right approach?  If so, are we also taking the same approach with gltf-pipeline?  If so, what about the ability to provide a single JSON config to create the pipeline stages?
* Why not have separate zip / unzip arguments instead of "doing the opposite."  Did you look at other command-line zip tools?


> Detects whether the tileset

Is this precise?  I would expect it to test each tile so, for example, if 3 of 10 tests are already zipped, it only zips the other 7.